web/js/components.js

```diff
@@ -36,6 +37,7 @@ class LrNav extends HTMLElement {
                     this.querySelector('.tabs').innerHTML += adminTabs
                 }
             }
+            setTimeout(()=>document.querySelector(`.tabs [href*="${location.href.split('/').pop()}"]`).classList.add("active"),0)
```

Comment:
Wow. this is the sort of thing that wins awards. One line, nearly completely unreadable, and wrapped in a lazy hack.